core/src/main/java/com/amazon/opendistroforelasticsearch/sql/planner/physical/ProjectOperator.java

```diff
@@ -60,11 +62,15 @@ public ExprValue next() {
     ImmutableMap.Builder<String, ExprValue> mapBuilder = new Builder<>();
     for (NamedExpression expr : projectList) {
       ExprValue exprValue = expr.valueOf(inputValue.bindingTuples());
-      // missing value is ignored.
-      if (!exprValue.isMissing()) {
-        mapBuilder.put(expr.getName(), exprValue);
-      }
+      mapBuilder.put(expr.getName(), exprValue);
     }
     return ExprTupleValue.fromExprValueMap(mapBuilder.build());
   }
+
+  @Override
+  public ExecutionEngine.Schema schema() {
+    return new ExecutionEngine.Schema(getProjectList().stream()
+        .map(expr -> new ExecutionEngine.Schema.Column(expr.getName(),
+            expr.getName(), expr.type())).collect(Collectors.toList()));
```

Comment:
Should we pass `expr.getAlias()` as second argument?

Reply:
Done.